src/kibana/components/vislib/visualizations/_point_series_chart.js

```diff
@@ -15,6 +15,30 @@ define(function (require) {
       PointSeriesChart.Super.apply(this, arguments);
     }
 
+    PointSeriesChart.prototype._stackMixedValues = function (stackCount) {
+      var currentStackOffsets = [0, 0];
+      var currentStackIndex = 0;
+
+      return function (d, y0, y) {
+        var firstStack = currentStackIndex % stackCount === 0;
+        var lastStack = ++currentStackIndex === stackCount;
+
+        if (firstStack) {
+          currentStackOffsets = [0, 0];
+        }
+
+        if (lastStack) currentStackIndex = 0;
+
+        if (y >= 0) {
+          d.y0 = currentStackOffsets[1];
+          currentStackOffsets[1] += y;
+        } else {
+          d.y0 = currentStackOffsets[0];
+          currentStackOffsets[0] += y;
```

Comment:
@panda01 note that it's tracking 2 different stacks as different indexes in that array - the first is the positive stack, the second is the negative one